README.CMake

```diff
@@ -179,7 +179,7 @@ Set GMT_RELEASE_PREFIX in cmake/ConfigUser.cmake and run cmake. Then do
 You should then edit ${GMT_RELEASE_PREFIX}/cmake/ConfigDefault.cmake and
 set GMT_PACKAGE_VERSION_MAJOR, GMT_PACKAGE_VERSION_MINOR, and
 GMT_PACKAGE_VERSION_PATCH. Also uncomment and set
-GMT_SOURCE_CODE_CONTROL_VERSION_STRING to the current svn version. Then
+GMT_SOURCE_CODE_CONTROL_VERSION_STRING to the current git version. Then
```

Comment:
But what would contain the GMT_SOURCE_CODE_CONTROL_VERSION_STRING? With SVN it was the revision number but in git we have only hash numbers.

Reply:
That is OK I think.  A release will say 6.0.0 but until then it says
pwessel@MacBeth:~-> gmt --version
6.0.0_r49baf49

Reply:
Right, but when it's defined (gmt_version.h.in) it falls to ``@GMT_PACKAGE_VERSION_WITH_SVN_REVISION@``

```
#ifdef GMT_SOURCE_CODE_CONTROL_VERSION_STRING
#	define GMT_STRING "@GMT_PACKAGE_VERSION_WITH_SVN_REVISION@ (r@GMT_SOURCE_CODE_CONTROL_VERSION_STRING@)"
```

Reply:
Yes, we need to make some edits there as well.